examples/energy_mng/src/controllers/weather.rs

```diff
@@ -0,0 +1,131 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+use crate::ui::*;
+use chrono::prelude::*;
+use core::cmp::Ordering;
+use slint::*;
+use std::rc::Rc;
+use weer_api::*;
+
+const WEATHER_API_KEY: &str = "WEATHER_API";
+const LAT_BERLIN: f32 = 52.520008;
+const LONG_BERLIN: f32 = 13.404954;
```

Comment:
Could that be obtained from a env variable as well?  (with a default to Berlin.)


For example, when we show a demo in, say, Nürnberg, we would like to put our current location there.

Reply:
That's a good idea, I will change it that way.

Reply:
In addition, could be a setting in the settings,  but that's for later as it would also need a keyboard